R/readWQPqw.R

```diff
@@ -33,7 +33,8 @@
 #' numerics based on a standard algorithm. If false, everything is returned as a character.
 #' @param ignore_attributes logical to choose to ignore fetching site and parameter
 #' attributes. Default is \code{FALSE}.
-#' @param legacy Logical. If TRUE, use legacy WQP services. Default is FALSE.
+#' @param legacy Logical. If TRUE, use legacy WQP services. Default is TRUE.
+#' Setting legacy = FALSE uses WQX3 services, which are in-development, use with caution.
```

Comment:
Oh, I like this message here. But I see. This one is much more visible than the constructNWISURL() function.